pgbouncer/README.md

```diff
@@ -0,0 +1,78 @@
+# PgBouncer Sample App
```

Comment:
Would be best to mention which platforms are supported (Linux, Windows, x86, arm64) and specifically for which platform is this Readme for